templates/SilverStripe/ElementalBannerBlock/Block/BannerBlock.ss

```diff
@@ -1,27 +1,32 @@
-<%-- Display the image (File) --%>
-<% if $File %>
-    <span class="banner-element__image">
-        $File
-    </span>
-
-<% end_if %>
-<div class="banner-element__content">
-    <% if $Title && $ShowTitle %>
-        <h2 class="banner-element__title">$Title</h2>
-    <% end_if %>
+<a class="banner-block"
+  <% if $CallToActionLink.Page.Link %>
+  <% with $CallToActionLink %>
+  href="{$Page.Link}"
+  <% if $TargetBlank %>target="_blank"<% end_if %>
+  <% if $Description %>title="{$Description.ATT}"<% end_if %>
+  <% end_with %>
+  <% end_if %>
+>
+<span class="banner-block__flex-box" role="banner">
+    <span class="banner-block__content">
+        <% if $Title && $ShowTitle %>
+            <span role="heading" class="banner-block__title h4">$Title</span>
+        <% end_if %>
+        <p>$Content.Summary(30)</p>
```

Comment:
`Summary()` takes *words*, which can still blow out the content depending how long those words are. That's a particular worry in french ;) 🇫🇷 

Also, this will cut off without any visible warning to the author, it looks like the system swallowed half of the text even though it's visible in the CMS. Was there a reason you didn't use a CSS ellipsis for this? 

Both cutting off words in PHP and ellipsis are flawed, since they happen after the authoring step, and "break" sentences. But in the case of ellipsis, at least it's clear what's happening, and the author can adjust.

Reply:
I had a look at using `text-overflow: ellipsis` however that only works when you set `white-space: nowrap;` so all your text appears on one line. Which is not going to work for our use case.

Reply:
OK, good enough for the demo - thanks for providing that context.